test/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -651,7 +776,7 @@ TEST(nntrainer_capi_nnmodel, getLayer_03_n) {
 /**
  * @brief Neural Network Model Get Layer Test
  */
-TEST(nntrainer_capi_nnmodel, getLayer_04_p) {
+TEST(nntrainer_capi_nnmodel, getLayer_06_n) {
```

Comment:
This unittest has same effect with getLayer_04_n

Reply:
okay